.gitignore

```diff
@@ -0,0 +1,5 @@
+#Node Modules
+node_modules
+
```

Comment:
I am putting this again actually make sure to add .idea folder here. :smile:

Reply:
Thanks, I Just added the .idea folder in the .gitignore file